HarpGuru/HarpGuru/index.tsx

```diff
@@ -32,28 +26,26 @@ const initialHarpStrataProps: HarpStrataProps = {
 const initialHarpStrata: HarpStrata = getHarpStrata(initialHarpStrataProps)
 const { Degree: initialDisplayMode } = DisplayModes
 
+const Stack = createStackNavigator()
+
 export const HarpGuru = (): ReactElement => {
   const [ activeHarpStrata, setActiveHarpStrata ] = useState(initialHarpStrata)
   const [ activeDisplayMode, setDisplayMode ] = useState(initialDisplayMode)
 
   const harpFaceProps: HarpFaceProps = { harpStrata: activeHarpStrata, displayMode: activeDisplayMode }
-
-  const displayModeTogglerProps = { setDisplayMode }
-
-  const firstPozitionButtonProps = { id: PozitionIds.First, activeHarpStrata, setActiveHarpStrata }
-  const secondPozitionButtonProps = { id: PozitionIds.Second, activeHarpStrata, setActiveHarpStrata }
-
-  const keyCHarpButtonProps = { id: PitchIds.C, activeHarpStrata, setActiveHarpStrata }
-  const keyDHarpButtonProps = { id: PitchIds.D, activeHarpStrata, setActiveHarpStrata }
+  const harpGuruHomeProps = {...harpFaceProps, setDisplayMode}
+  const harpStrataControlProps = { activeHarpStrata, setActiveHarpStrata }
 
   return (
-    <View style={styles.guru}>
-      <HarpFace {...harpFaceProps} />
-      <DisplayModeToggler {...displayModeTogglerProps} />
-      <PozitionButton {...firstPozitionButtonProps} />
-      <PozitionButton {...secondPozitionButtonProps} />
-      <PitchButton {...keyCHarpButtonProps} />
-      <PitchButton {...keyDHarpButtonProps} />
-    </View>
+    <NavigationContainer>
+      <Stack.Navigator>
+        <Stack.Screen name='HarpGuruHome'>
+          {(): ReactElement => <HarpGuruHome {...harpGuruHomeProps} />}
+        </Stack.Screen>
+        <Stack.Screen name='PozitionControlPanel'>
+          {(): ReactElement => <PozitionControlPanel {...harpStrataControlProps} />}
+        </Stack.Screen>
+      </Stack.Navigator>
+    </NavigationContainer>
```

Comment:
It seems like it's preferable not to pass any properties down to the screens like this, but to rather create some contexts. This will result in testing for those screens to be more difficult, but we just need to make sure that the props are passed down from there rather than the context used.

Reply:
Improved by passing down to screen components which all take the same props.